pkg/registry/credentials.go

```diff
@@ -1,44 +1,28 @@
 package registry
 
 import (
-	"github.com/docker/cli/cli/config"
 	"github.com/docker/cli/cli/config/configfile"
 	"github.com/docker/cli/cli/config/credentials"
 	"oras.land/oras-go/v2/registry/remote/auth"
 )
 
-// CredentialStore for registry credentials such as ~/.docker/config.json.
-type CredentialStore struct {
-	directory  string
+// DockerCredentialStore for Docker registry credentials, like ~/.docker/config.json.
```

Comment:
This is all fine, but I don't like docker in the name. I was trying to hide that. We don't have to be stuck with docker forever.